### ISSUE TITLE
com.google.playservices has been renamed to cordova-plugin-googleplayservices

### ISSUE THREAD

**Issue by CoHyper** [danwilson/google-analytics-plugin#96]
Installing "com.danielcwilson.plugins.googleanalytics" for android
WARNING: com.google.playservices has been renamed to cordova-plugin-googleplayservices. You may not be getting the latest version! We suggest you `cordova plugin rm com.google.playservices` and `cordova plugin add cordova-plugin-googleplayservices`.
Fetching plugin "com.google.playservices@19.0.0" via cordova plugins registry

**Comment by bdusell** [comment]
I'm running into this as well. It's actually breaking my build process, since version 19.0.0 has apparently been taken down. Try installing the package and you'll get this:

```
% npm install com.google.playservices@19.0.0
npm http GET https://registry.npmjs.org/com.google.playservices/19.0.0
npm http 404 https://registry.npmjs.org/com.google.playservices/19.0.0
npm ERR! Error: version not found: 19.0.0 : com.google.playservices/19.0.0
npm ERR!     at RegClient.<anonymous> (/usr/share/npm/node_modules/npm-registry-client/lib/request.js:272:14)
npm ERR!     at Request.self.callback (/usr/lib/nodejs/request/index.js:148:22)
npm ERR!     at Request.EventEmitter.emit (events.js:98:17)
npm ERR!     at Request.<anonymous> (/usr/lib/nodejs/request/index.js:896:14)
npm ERR!     at Request.EventEmitter.emit (events.js:117:20)
npm ERR!     at IncomingMessage.<anonymous> (/usr/lib/nodejs/request/index.js:847:12)
npm ERR!     at IncomingMessage.EventEmitter.emit (events.js:117:20)
npm ERR!     at _stream_readable.js:920:16
npm ERR!     at process._tickCallback (node.js:415:13)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>
```

I won't be able to continue using this plugin until this dependency has been updated.

**Comment by danwilson** [comment]
@bdusell `npm install` is not what you would want here. `cordova plugin add com.google.playservices@19.0.0` is correct.

All plugins are moving registries, and this plugin depends on the play services plugin.  The current version is valid for now and still installs, but I will leave this ticket open since it needs to be updated.

**Comment by Daz66** [comment]
Hi. Is there any solution for this yet? My build has been trashed for ages too - it may be valid to use v19 but that corrupts any other plugins that use the latest. Unless there's a fix it looks like I won't be able to use your solution. Help?!

Thanks for any advice, Daz

**Comment by danwilson** [comment]
Closed with #118

**Comment by SanderPs** [comment]
`cordova plugin add com.google.playservices@19.0.0` did not work for me. Or any other version.

I solved the problem with:
`cordova plugin add cordova-plugin-googleplayservices`
